### DISCUSSION TITLE
SQL Server -> Tables criadas erroneamente na master

### DISCUSSION THREAD

**Discussion by pedro-mileipp** [balta-io/forum#148] [Tira Dúvidas] [answered]
- Bom dia, caríssimos. Criei erroneamente as tables na master, porém não consigo apagá-las, poderiam me ajudar?

**Comment by AndreBarboza17** [comment, marked as answer]
Olá bom dia.

Já tentou realizar o comando DROP TABLE [NOME_DA_TABELA] no banco master?
